test/Analysis/test-allocation.mlir

```diff
@@ -232,34 +232,34 @@ tt.func @multi_color(%A : !tt.ptr<f16>) {
   %10 = ttg.local_load %cst_7 : !ttg.memdesc<2x32xf16, #A_SHARED, #ttg.shared_memory, mutable> -> tensor<2x32xf16, #AL>
   %cst_12 = arith.constant dense<0.000000e+00> : tensor<4x16xf16, #AL>
   %cst_13 = arith.constant dense<0.000000e+00> : tensor<8x32xf16, #AL>
-  // CHECK-NEXT: size = 1920
+  // CHECK-NEXT: size = 1504
   tt.return
 }
 
 // This example triggers graph coloring with multiple rounds
 // CHECK-LABEL: multi_color_multi_rounds
 tt.func @multi_color_multi_rounds(%arg0: !tt.ptr<f16>) {
```

Comment:
Can you add a check for total smem usage to these tests, and give a summary of how they've changed?

Reply:
The total smem usage is denoted on the last line

Reply:
Oh awesome, so it's all wins then.